public/app/features/browse-dashboards/components/BrowseActions/DeleteModal.tsx

```diff
@@ -27,7 +27,7 @@ export const DeleteModal = ({ onConfirm, onDismiss, selectedItems, ...props }: P
         folder: Object.keys(selectedItems.folder).length,
       },
       source: 'browse_dashboards',
-      restore_enabled: config.featureToggles.dashboardRestore,
+      restore_enabled: config.featureToggles.dashboardRestore ?? false,
```

Comment:
```suggestion
      restore_enabled: Boolean(config.featureToggles.dashboardRestore),
```
Could we rap it into `Boolean` instead?

Reply:
@Clarity-89, is using `Boolean` a better practice? 🤔

Reply:
It just looks cleaner imo.